Source/WebCore/platform/graphics/ColorConversion.cpp

```diff
@@ -42,8 +42,13 @@ LCHLike convertToPolarForm(const LabLike& color)
     // https://drafts.csswg.org/css-color/#lab-to-lch
     auto [lightness, a, b, alpha] = color.resolved();
 
-    float hue = rad2deg(atan2(b, a));
+    // Epsilon chosen to ensure `white` (the SRGB named value) is
+    // considered achromatic (e.g. we set hue to NaN) when converted
+    // to lch/oklch.
+    static constexpr auto epsilon = 0.02f;
+
     float chroma = std::hypot(a, b);
+    float hue = (std::abs(a) < epsilon && std::abs(b) < epsilon) ? std::numeric_limits<float>::quiet_NaN() : rad2deg(atan2(b, a));
```

Comment:
Does this use the `double` versions of `rad2deg` and `atan2`? If so, is that better than using the `float` versions?

Reply:
I'm not sure, but I am going to be doing an experiment of making all the conversion code use double for everything, delaying conversion back to float until all substeps / transforms are completed, so it will be a good opportunity to look into this.